ch_tools/common/config.py

```diff
@@ -46,6 +46,16 @@
             },
         },
     },
+    "monitoring": {
+        "output": {
+            "escaping_rules": [
+                {
+                    "pattern": "\n",
+                    "replacement": "",
```

Comment:
Is any sample with EOL in output text? May be better make replacement with some delimiter (space) to change
```
foo
bar
```
on
```
foo bar
```
not
```
foobar
```
?

Reply:
Good idea! I've updated the replacement string to `" "`.